backend/controllers/searchExternalUsers.js

```diff
@@ -5,40 +5,35 @@ const mapUsernameAndEmail = (u) => {
 
 const searchFactory = (
   paginationService,
-  getAssignableGroupsOrPrisonsApi,
+  getAssignableGroups,
   getSearchableRolesApi,
   searchApi,
   pagingApi,
   searchUrl,
   maintainUrl,
   searchTitle,
-  dpsSearch,
   allowDownload,
 ) => {
   const index = async (req, res) => {
-    const [groupOrPrisonDropdownValues, searchableRoles] = await Promise.all([
-      getAssignableGroupsOrPrisonsApi(res.locals),
+    const [groupDropdownValues, searchableRoles] = await Promise.all([
+      getAssignableGroups(res.locals),
       getSearchableRolesApi(res.locals),
     ])
     const roleDropdownValues = searchableRoles.map((r) => ({
       text: r.roleName,
       value: r.roleCode,
     }))
-    const showGroupOrPrisonDropdown = Boolean(res.locals?.user?.maintainAccessAdmin || !dpsSearch)
 
-    res.render('search.njk', {
+    res.render('searchExternalUsers.njk', {
       searchTitle,
       searchUrl,
-      groupOrPrisonDropdownValues,
+      groupDropdownValues,
       roleDropdownValues,
-      showGroupOrPrisonDropdown,
```

Comment:
Always show group dropdown values for external search (code used to be: `const showGroupOrPrisonDropdown = Boolean(res.locals?.user?.maintainAccessAdmin || !dpsSearch)`)